src/solidity-frontend/solidity_convert.cpp

```diff
@@ -1618,11 +1651,12 @@ bool solidity_convertert::get_conditional_operator_expr(
 
 bool solidity_convertert::get_cast_expr(
   const nlohmann::json &cast_expr,
-  exprt &new_expr)
+  exprt &new_expr,
+  const nlohmann::json &int_literal_type = nullptr)
```

Comment:
default parameter value should be in the header, not .cpp.

Reply:
Also, it's better to initialize const ref to refer to a valid object. If this parameter needs to be nullptr by default, we shall use a const pointer instead of const ref.

Reply:
Thanks for the suggestion. I will use a const pointer